ucm2/Qualcomm/sc7180/rt5682-max98357a/HiFi.conf

```diff
@@ -0,0 +1,99 @@
+# Use case configuration for ALC5682+MAX98357A on SC7180
+
+SectionVerb {
+	Value {
+		TQ "HiFi"
+	}
+}
+
+SectionDevice."Speaker" {
+	Comment "Speaker"
+
+	Value {
+		PlaybackChannels 2
+		PlaybackPCM "hw:${CardId},1"
+		PlaybackPriority 100
+		PlaybackMixerElem "Speaker"
+		PlaybackVolume "Speaker Playback Volume"
+	}
+}
+
+SectionDevice."Headphones" {
+	Comment "Headphones"
+
+	EnableSequence [
+		cset "name='HPOL Playback Switch' 1"
+		cset "name='HPOR Playback Switch' 1"
+		cset "name='Stereo1 DAC MIXL DAC L1 Switch' 1"
+		cset "name='Stereo1 DAC MIXR DAC R1 Switch' 1"
+	]
+
+	DisableSequence [
+		cset "name='HPOL Playback Switch' 0"
+		cset "name='HPOR Playback Switch' 0"
+		cset "name='Stereo1 DAC MIXL DAC L1 Switch' 0"
+		cset "name='Stereo1 DAC MIXR DAC R1 Switch' 0"
+	]
+
+	Value {
+		PlaybackPCM "hw:${CardId},0"
+		PlaybackPriority 200
+		PlaybackMasterElem "DAC1"
+		PlaybackMixerElem "Headphone"
+		PlaybackVolume "Headphone Playback Volume"
+		JackHWMute "Speaker"
+	}
+}
+
+SectionDevice."Mic" {
+	Comment "Internal Microphone"
+
+	EnableSequence [
+		cset "name='IF1 01 ADC Swap Mux' 1"
```

Comment:
It seems that Mic and Headphones mic conflicts ? Add ConflictingDevice section to one device (the alsa-lib will add it for the other automatically).